CMakeLists.txt

```diff
@@ -111,9 +112,9 @@ if(OSI_BUILD_FLATBUFFER)
     set(fbs "${proto_base}.fbs")
     add_custom_command(
       OUTPUT "${fbs}"
-      COMMAND $<TARGET_FILE:flatc> -I "${PROTOBUF_IMPORT_DIRS}" -o "${CMAKE_CURRENT_BINARY_DIR}" --proto "${CMAKE_CURRENT_SOURCE_DIR}/${proto}"
-      DEPENDS "${proto}" flatc
-      WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+      COMMAND ${FLATBUFFERS_FLATC_EXECUTABLE} -I "${PROTOBUF_IMPORT_DIRS}" -I "${CMAKE_CURRENT_BINARY_DIR}" -o "${CMAKE_CURRENT_BINARY_DIR}" --proto "${proto}"
```

Comment:
Is this change fixing an issue?

Reply:
Yes: 
- Use of -I due to the cleaner separation of SOURCE and BINARY directories (we formerly generated files in the SOURCE directory, which we shouldn't)
- Since flatbuffers are not longer co-built, the flatc executable is not a target anymore, but rather referenced via variable.